app/Jobs/BookReturnReminderJob.php

```diff
@@ -34,16 +36,47 @@ public function __construct()
     public function handle()
     {
         $ledges = Ledge::all();
-        foreach ($ledges as $ledge) {
-            if ($ledge->return_date !== NULL && $this->differenceBetweenTodayAndDate($ledge->return_date) === 7)
-            {
-                Mail::to($ledge->borrower->email)->send(new BookReturnReminderMail($ledge));
-            }
+        foreach ($ledges as $ledge)
+        {
+            $this->sendReminderMail($ledge);
         }
     }
 
-    private function differenceBetweenTodayAndDate($date)
+    private function sendReminderMail(Ledge $ledge)
+    {
+        if ($ledge->status === LedgeStatus::WaitingPickup && $ledge->return_date !== NULL && $this->differenceInDaysBetweenTodayAndDate($ledge->return_date) === 1)
```

Comment:
There are two things to mention here:
- [ ] we should use the pickup_date not the return date 
- [ ] We should send a different email (as this is a reminder to pick the book up, not the one to return it. I am ok if you just make a copy of the return for now), but they have to be different as they will have quite a different language